shared/src/main/scala/amf/core/remote/Vendor.scala

```diff
@@ -2,7 +2,6 @@ package amf.core.remote
 
 import scala.scalajs.js.annotation.{JSExport, JSExportAll, JSExportTopLevel}
 
-@JSExportAll
```

Comment:
`unapply` method can't be exported because it returns an Option. It shouldn't return a ClientOption because it is used internally.